.github/workflows/main.yml

```diff
@@ -349,14 +302,16 @@ jobs:
     - run: |
         cargo build -p wasm-bindgen-cli
         ln -snf `pwd`/target/debug/wasm-bindgen $(dirname `which cargo`)/wasm-bindgen
-    - run: mv _package.json package.json && npm install && rm package.json
+    - run: mv _package.json package.json && corepack pnpm install && rm package.json
```

Comment:
I left the `packageManager` field out of the `package.json` files compared to the commit in #4047 so it always uses the latest one. It prints this in the CI now, which I think is ok:
> The local project doesn't define a 'packageManager' field. Corepack will now add one referencing [current version]